oeps/oep-0001.rst

```diff
@@ -7,7 +7,7 @@ OEP-1: OEP Purpose and Guidelines
 +---------------+-------------------------------------------+
 | Title         | OEP Purpose and Guidelines                |
 +---------------+-------------------------------------------+
-| Last-Modified | 2016-06-24                                |
+| Last-Modified | 2016-08-08                                |
 +---------------+-------------------------------------------+
 | Author        | Calen Pennington <cale@edx.org>           |
```

Comment:
I think pluralizing Author to Authors and adding Joel's name makes sense. We should leave Cale's name first as the original author.